### ISSUE TITLE
Support for Normal Maps in 3dsMaxPhysicalMaterial.cpp

### ISSUE THREAD

**Issue by WGeeBalti** [facebookincubator/FBX2glTF#254]
Am I missing something or is this currently missing support for passing along the normal map for 3DS Max Physical Materials? 

This seems present in Stingray and the Standard Material versions and should be roughly the same process as passing along the other maps, like: 

const auto* baseTex = getTex("base_color");
add> const auto* normalTex = getTex("normal");
...
res->texBaseColor = baseTex;
add> res->texNormal = normalTex ;

Any help would be greatly appreciated!

**Comment by MukarramSC** [comment]
Yup. support for normal maps is missing. I've modified a local copy of the code to add support for that. Never got the time to do it the right way (i-e make a pull request out of it). Let me know if you need the binary with support for normal maps.

**Comment by WGeeBalti** [comment]
Thanks! I hacked in a solution for normals and AO maps as well. Mine is not general-purpose enough for a PR either, but I appreciate the response :-)

**Comment by MukarramSC** [comment]
Please correct me if I'm wrong, but I think the Physical material doesn't carry over alpha/opacity/transparency as well. Ever encountered such issue?

**Comment by WGeeBalti** [comment]
We haven't had issue with alpha - we typically use png textures with an
alpha channel if we need transparency.
